client/acct_mgr.cpp

```diff
@@ -884,6 +894,18 @@ int ACCT_MGR_INFO::write_info() {
             no_project_notices?1:0,
             dynamic?1:0
         );
+        if (strlen(user_name)) {
+            string x = user_name;
+            xml_unescape(x);
```

Comment:
At this point you are writing the user_name and team_name to the XML document.  Don't you want to use xml_escape not xml_unescape?

Reply:
See below